modular_zubbers/code/modules/antagonists/bloodsucker/bloodsuckers/bloodsucker.dm

```diff
@@ -153,8 +149,7 @@
 /datum/antagonist/bloodsucker/remove_innate_effects(mob/living/mob_override)
 	. = ..()
 	var/mob/living/carbon/current_mob = mob_override || owner.current
-	remove_signals_from_heart(current_mob)
-	UnregisterSignal(current_mob, list(COMSIG_LIVING_LIFE, COMSIG_ATOM_EXAMINE, COMSIG_LIVING_DEATH, COMSIG_SPECIES_GAIN, COMSIG_QDELETING))
+	UnregisterSignal(current_mob, list(COMSIG_LIVING_LIFE, COMSIG_ATOM_EXAMINE, COMSIG_LIVING_DEATH, COMSIG_SPECIES_GAIN, COMSIG_QDELETING, COMSIG_ENTER_COFFIN, COMSIG_MOB_STAKED))
```

Comment:
```suggestion
	UnregisterSignal(current_mob, list( \
					COMSIG_LIVING_LIFE, \
					COMSIG_ATOM_EXAMINE, \
					COMSIG_LIVING_DEATH, \
					COMSIG_SPECIES_GAIN, \
					COMSIG_QDELETING, \
					COMSIG_ENTER_COFFIN, \
					COMSIG_MOB_STAKED \
					))
```

Reply:
List doesn't need the \, 99% sure

Reply:
Oh yeah, either way don't leave it on a signle line as it's hard to read